app/views/spree/components/products/_product_form.html.erb

```diff
@@ -0,0 +1,63 @@
+<%
+  html = {
+    itemprop: 'offers',
+    itemscope: true,
+    itemtype: 'http://schema.org/Offer'
+  }
+%>
+
+<%= form_for :order, url: populate_orders_path, html: html do |f| %>
+  <%= render partial: 'spree/components/products/product_variants' %>
+
+  <% if @product.price_for(current_pricing_options) and !@product.price.nil? %>
+    <div class="product-submit" id="product-price">
+      <h2 class="product-submit__title">
+        <span
```

Comment:
What about moving the price markup into a specific component?

Reply:
Yes, way better!